gcc/rust/expand/rust-macro-builtins-asm.h

```diff
@@ -8,6 +8,9 @@
 #include "system.h"
 namespace Rust {
 
+std::string
+strip_double_quotes (const std::string &str);
```

Comment:
I think this function is used privately, so it does not need to be declared here. feel free to disregard if that's not the case

Reply:
resolved